package.json

```diff
@@ -57,6 +57,11 @@
     "sass-loader": "^4.0.0",
     "style-loader": "^0.13.1",
     "underscore": "^1.8.3",
-    "webpack": "^1.13.1"
+    "webpack": "^1.13.1",
+    "yelp-fusion": "^3.0.0"
+  },
+  "devDependencies": {
+    "concurrently": "^4.1.0",
+    "nodemon": "^1.19.0"
```

Comment:
I think it should be up to the dev to install `nodemon` globally on their system rather than have it listed as a dev dependency. It's a pretty big package and isn't necessary to run the app. For example, I use VS Code's debugger to run most of my node apps.